### PR TITLE
Bubble arrow left (and other directions)

### PR DESCRIPTION
**Pull request by matikrk**
Possible direction left/right/top/button
Placing arrow on start/middle/end of line

example
screen shot 2017-06-05 at 16 02 23

### CODE REVIEW

**Comment by matzimowski** [comment]
Cool, could you provide screenshots, please?

**Comment by burakukula** [comment]
@matikrk screenshots please

File: src/components/bubble/_bubble.scss [hunk 1]
```diff
@@ -19,28 +19,81 @@ $includeHtml: false !default;
...
 
-    &::before {
+    @mixin arrow($arrowSize: gutter(0.5)) {
+      border: $arrowSize solid transparent;
       content: '';
-      border: $triangleSize solid transparent;
-      border-bottom-color: $bubbleBorderColor;
       position: absolute;
-      top: -(2 * $triangleSize);
+      top: 0;
```
Comment (aju) [inline src/components/bubble/_bubble.scss]: This looks like it will break backward compatibility :}

**Comment by burakukula** [comment]
@matikrk please fix 'full and grow' example - there is one that don't have any triangle

File: src/components/bubble/_bubble.scss [hunk 2]
```diff
@@ -19,28 +19,81 @@ $includeHtml: false !default;
     padding: gutter(0.5) gutter(1);
     position: relative;
 
-    &::before {
+    @mixin arrow($arrowSize: gutter(0.5)) {
```
Comment (vergilius) [inline src/components/bubble/_bubble.scss]: shouldn't mixin be moved up above block?

Comment (matikrk) [inline src/components/bubble/_bubble.scss]: done

**Comment by matikrk** [comment]
Examples updated
New screenshot added
Deleted  --full-and-grown

**Comment by kdzwinel** [comment]
👏 Congrats on a first PR!

**Pull request merged**